exercises/delia_solution/teenager/teenager.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"unicode"
+)
+
+func bobAnswer(question string) string {
+	answer := ""
+	if IsQuestion(question) {
+		if IsUpperCase(question) {
+			answer = "Calm down, I know what I'm doing!"
+		} else {
+			answer = "Sure."
+		}
+	} else if IsUpperCase(question) {
+		answer = "Whoa, chill out!"
+	} else if question == "" {
+		answer = "Fine. Be that way!"
+	} else {
+		answer = "Whatever."
+	}
+	return answer
+}
+
```

Comment:
Could use a switch here instead of nested ifs

Reply:
I think the function is the same in this case, personally I'm not a fan of switch structure and there aren't a lot of cases to write a switch